index.bs

```diff
@@ -485,23 +485,19 @@ The header's ABNF is:
   Sec-CH-UA = sf-list
 ```
 
-To <dfn abstract-op local-lt="set-ua">return the `Sec-CH-UA` value for a request</dfn>, [=user agents=] MUST:
-
-1.  Let |list| be a [=/list=], initially empty.
-
-2. For each |brandVersion| in [=user agent/brands=]:
+To <dfn abstract-op>return the `Sec-CH-UA` value for a request</dfn>,
+[=user agents=] MUST:
 
+1. Let |list| be a [=/list=], initially empty.
+1. Let |brands| be the result of running the [=create brands=] steps, with
+    [=user agent/significant version=] passed as an argument.
+1. For each |brand| in |brands|:
     1. Let |parameter| be a [=dictionary=], initially empty.
-
-    2. Set |parameter|["param_name"] to "v".
-
-    3. Set |parameter|["param_value"] to |brandVersion|'s {{NavigatorUABrandVersion/version}}.
-
-    2. Let |pair| be a tuple comprised of |brandVersion|'s {{NavigatorUABrandVersion/brand}} and |parameter|.
-
-    3. Append |pair| to |list|.
-
-3. Return the output of running [=serializing a list=] with |list| as input.
+    1. Set |parameter|["param_name"] to "v".
```

Comment:
I think the serialization can be moved to a helper algorithm as well, to avoid duplicating it in both places.

Reply:
Addressed in https://github.com/WICG/ua-client-hints/pull/250/commits/01cc7d30ef0ab38952cfdea275dcd7504a8f3aa4.